README.md

```diff
@@ -16,8 +16,9 @@ For further developer and user documentation please visit [the wiki](https://git
 ##### Minimum PHP Version
 * PHP >= 5.4
 
-##### Minimum ownCloud Version
-* >= 8.1
+##### Minimum ownCloud / Nextcloud Version
```

Comment:
I agree. If it runs - fine. But otherwise it could mean there is complete official support for ownCloud (and i don't think there is, is? ;-) )